src/middleware/cache.js

```diff
@@ -1,5 +1,5 @@
 module.exports = (req, res, next) => {
-	res.FT_NO_CACHE = 'max-age=0, no-cache, no-store, must-revalidate';
+	res.FT_NO_CACHE = 'max-age=0, no-cache, must-revalidate';
```

Comment:
I'm sure there's some danger here. @commuterjoy was always very opinionated about really strict non-caching

Reply:
Yep I'm sure there's a reason, but don't know what it is.

Personally I don't see an issue with 304 responses, but willing to be shown otherwise.

Reply:
I think it's due to intermediate caches not necessarily respecting vary, so we need to force intermediates not to store, and no-store is the only bulletproof way to do so

Reply:
I think it's safe to remove, I see other sites have simplified their no-cache directives too. IIRC the original FT no cache was based on a bit of an audit of what other personalised sites do. If you look at github, google etc. it's often just `max-age=0` these days.